src/ert/scheduler/slurm_driver.py

```diff
@@ -367,6 +369,18 @@ async def _poll_once_by_scontrol(
         ) and missing_job_id in self._scontrol_cache:
             return self._scontrol_cache[missing_job_id]
 
+        info = await self._run_scontrol(missing_job_id)
+        if info is None:
+            logger.error("scontrol failed, trying sacct")
```

Comment:
This is probably a `logger.warning` since we have a fallback.

Reply:
I agree, I will turn some errors into warnings.